### ISSUE TITLE
Reinitialising Client leads to double reporting on Android but not iOS

### ISSUE THREAD

**Issue by nabati** [bugsnag/bugsnag-react-native#335]
AFAIK, currently, there's no way of setting the `codeBundleId` dynamically at a later point in time on the client. This is needed because the codeBundleId might be known asynchronously and not at startup. 

We went around this by initialising a new client whenever the codeBundleId is updated (or known). 

The issue with this is that when we initialise a new client  at a later stage, the old client is still active and keeps reporting bugs but with the old codeBundleId. 

This behaviour only shows on Android, it doesn't seem to do double reporting on iOS.

Running 
```
    "bugsnag-react-native": "2.16.0",
```

Basically: 

```
new Client (someConfiguration) // codeBundleId: 'A'
// A minute later
new Client (otherConfiguration) // codeBundleId: 'B'
....
throw new Error('testerror') // Will report two errors with codeBundleId A and B on Android, but not on iOS.    
```

How do you work around this limitation? 
Why is there no `setCodeBundleId` in the client? 
Why is there no way discard or stop a client?

**Comment by phillipsam** [comment]
We have released a brand new version of the React Native notifier as part of the `bugsnag-js` repository: [v7.3.0](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.3.0).

This issue is no longer applicable in that new version. Let us know if you have further questions.
